app/src/main/kotlin/dev/patrickgold/florisboard/app/settings/localization/LocalizationScreen.kt

```diff
@@ -58,6 +66,8 @@ fun LocalizationScreen() = FlorisScreen {
     val keyboardManager by context.keyboardManager()
     val subtypeManager by context.subtypeManager()
     val cacheManager by context.cacheManager()
+    var chosenSubtypeToDelete: Subtype? by rememberSaveable { mutableStateOf(null) }
```

Comment:
We need to use a custom saver, because the default one crashes the app on rotation.
```suggestion
    var chosenSubtypeToDelete: Subtype? by rememberSaveable(saver = SubtypeSaver) { mutableStateOf(null) }
```